features/feelingMyself/index.js

```diff
@@ -0,0 +1,29 @@
+const replies = require('./replies.js')
+const {
+  randomArrayItem,
+  messageFromAnyThread,
+  messageFromGeneralChannel,
+} = require('../../lib')
+
+const shouldMessage = (message) => {
+  const notFromThread = !messageFromAnyThread(message)
+  const fromAllowableChannel = messageFromGeneralChannel(message)
+
+  return fromAllowableChannel && notFromThread
+}
+
+const REGEXP = /\bfeel(ing)? myself\b/ig
```

Comment:
We could potentially also add "feels(ing) herself/himself" here, but in that case we would have to complicate the logic for replying ("I bet he is!"), so I think that's fine to go with `myself` for MVP 🙂

Reply:
Technically, we can use: `/feel(?:s|ing)? (my|her|him|our|your|them)sel(?:f|ves)/` and then extract and analyze the extracted group afterwards :thinking:. I think we can add it before merge (along with tests, of course)